cross-media-measurement/src/main/cc/wfa/measurement/common/crypto/liquid_legions_v2_encryption_utility.cc

```diff
@@ -802,17 +798,30 @@ CompleteExecutionPhaseTwoAtAggregator(
 
   // Estimates reach.
   int64_t active_register_count = tuple_counts - blinded_histogram_noise_count;
-  if (request.has_noise_baseline()) {
+  if (request.has_reach_dp_noise_baseline()) {
     auto options = GetGlobalReachDpNoiseOptions(
-        request.noise_baseline().global_reach_dp_noise(),
-        request.noise_baseline().contributors_count());
-    int global_reach_dp_noise_baseline = options.shift_offset * options.num;
+        request.reach_dp_noise_baseline().global_reach_dp_noise(),
+        request.reach_dp_noise_baseline().contributors_count());
+    int64_t global_reach_dp_noise_baseline = options.shift_offset * options.num;
```

Comment:
Sorry I should've caught this earlier but this options.num should be changed to the total # of parties, not just the # of _honest_ parties. (This may not be important in actual deployment but probably is important for experiments/benchmarking.)

Reply:
In the experiment, we have 1 aggregator and 2 non-aggregator, and we assume # of honest parties = 3 = total # of parties. This is also the # of contributors to each noise. does this work?

Reply:
This discussion should probably be moved somewhere else, but shouldn't we at least experiment on the case where not all parties are honest for the paper?